src/styles/abstracts/_mixins.scss

```diff
@@ -44,3 +44,19 @@
   justify-content: center;
   align-items: center;
 }
+
+@mixin button-bg($bg) {
+  background: $bg;
+
+  padding: rem(5px 10px);
+  border-radius: rem(3px);
+  text-decoration: none;
```

Comment:
Since you named it `button-bg`, should we keep only the properties that are related to `background`. What do you think?

Reply:
At first, I thought these properties are related to `background`. What if I rename it to be `button`?